src/pages/RoomInvitePage.tsx

```diff
@@ -32,8 +32,9 @@ import {getLoginsByAccountIDs} from '@libs/PersonalDetailsUtils';
 import {addSMSDomainIfPhoneNumber, parsePhoneNumber} from '@libs/PhoneNumber';
 import type {MemberEmailsToAccountIDs} from '@libs/PolicyUtils';
 import {isPolicyEmployee as isPolicyEmployeeUtil} from '@libs/PolicyUtils';
+import {getReportName} from '@libs/ReportNameUtils';
```

Comment:
this is not correct. earlier the `getReportName` was using full  implementation to drive the report name but this one is different. 

I would suggest we don't change these functions and ignore errors by supressing deprecated errors. There is already someone working on this refactor.